TASK-3/BogusRepository.cs

```diff
@@ -0,0 +1,35 @@
+﻿using Bogus;
+
+namespace TASK;
+
+public class BogusRepository
+{
+    private enum _vehicleModels
+    {
+        BMW,
+        VW
+    };
```

Comment:
Enum should be in enum class, not in the bogus class

Reply:
Moved the enum to a separate file.